docs/search-service.md

```diff
@@ -0,0 +1,20 @@
+
+# Search service
+
+Handles all item searching.
+
+The implemetation is leveraging an Elasticsearch server to store and then search Items. To access Elasticsearch this service uses the [HTTP API](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs.html) so this is an example of wrapping a 3rd party HTTP API on a [Lagom Descriptor](http://www.lagomframework.com/documentation/).
```

Comment:
Is "Lagom Descriptor" meant to link to the documentation index? Looks like maybe it should go to http://www.lagomframework.com/documentation/1.2.x/java/ServiceDescriptors.html